codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithyjava/generator/awssdk/v2/ToDafnyAwsV2.java

```diff
@@ -705,7 +705,7 @@ MethodSpec generateConvertOpaqueServiceError() {
         "nativeValue"
       )
       .addStatement(
-        "return $T.create_Opaque(nativeValue)",
+        "return $T.create_OpaqueWithText(nativeValue, dafny.DafnySequence.asString(nativeValue.getMessage()))",
```

Comment:
Blocking: Regrettably, we are going to need both an option for OpaqueWithText and Opaque.

Though maybe not in ToDafnyAws,
it probably simplifies our complexity to just know it is ALWAYS an option.

Reply:
I don't understand. How can the return type be optional? Where do we put the configuration?